pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -147,6 +147,15 @@ type WorkerPool struct {
 	// KubernetesVersion is the kubernetes version in this worker pool
 	// +optional
 	KubernetesVersion *string `json:"kubernetesVersion,omitempty"`
+
+	// NodeTemplate contains resource information of the machine which is used by Cluster Autoscaler to generate nodeTemplate during scaling a nodeGroup from zero
+	NodeTemplate NodeTemplate `json:"nodeTemplate"`
```

Comment:
This field needs to be optional for backwards-compatibility (see https://github.com/gardener/gardener/blob/master/docs/development/changing-the-api.md https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md, https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md)
```suggestion
    // +optional
	NodeTemplate *NodeTemplate `json:"nodeTemplate,omitempty"`
```